homework-functions/index.js

```diff
@@ -0,0 +1,93 @@
+class TrieNode {
```

Comment:
please merge the master branch into yours, your PR shouldn't contain code from another task